sdk/communication/Azure.Communication.Common/src/Identity/ITokenCredential.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Azure.Communication.Identity
 {
-    internal interface IUserCredential : IDisposable
+    internal interface ITokenCredential : IDisposable
```

Comment:
We could rename this to `ICommunicationTokenCredential` to avoid confusing ourselves, but not too important as it's internal.

Reply:
Sounds good, I will address this in my next PR!